Common.mk

```diff
@@ -163,7 +165,9 @@ COMMA=,
 # $1 - release branch
 CACHE_IMPORT_IMAGES=$(foreach reg,$(PROD_ECR_REG) $(DEV_ECR_REG),type=registry$(COMMA)ref=$(reg)/$(call IF_OVERRIDE_VARIABLE,$(IMAGE_COMPONENT_VARIABLE),$(IMAGE_COMPONENT)):$(GIT_TAG)-eks-$(1)-latest)
 
-IMAGE_IMPORT_CACHE?=$(foreach branch,$(SUPPORTED_K8S_VERSIONS),$(if $(filter $(GIT_TAG),$(shell cat ./$(branch)/GIT_TAG)),$(call CACHE_IMPORT_IMAGES,$(branch)),))
+OLDEST_SUPPORTED_BRANCH=$(firstword $(SUPPORTED_K8S_VERSIONS))
```

Comment:
I think wwe still want what we had before along with this. Find the oldest supported branch with the same git_tag.